charts/fission-all/templates/post-install-job.yaml

```diff
@@ -22,15 +22,20 @@ spec:
       labels:
         release: {{ .Release.Name }}
         app: {{ template "name" . }}
-      annotations:
-        {{- if .Values.enableIstio }}
-        "sidecar.istio.io/inject": "false"
-        {{- end }}
```

Comment:
What is the intention of removing this one?

Reply:
Added back, as I don't see any value of Istio injection in these post hooks.